test/functional/mempool_limit.py

```diff
@@ -60,22 +67,29 @@ def run_test(self):
                 node, txouts, 25, batch_of_txid * base_fee
             )
 
-        self.log.info('The tx should be evicted by now')
+        self.log.info("The tx should be evicted by now")
         # The number of transactions created should be greater than the ones present in the mempool
         assert_greater_than(no_of_large_tx_sent, len(node.getrawmempool()))
         # Initial tx created should not be present in the mempool anymore as it had a lower fee rate
         assert tx_to_be_evicted_id not in node.getrawmempool()
 
-        self.log.info('Check that mempoolminfee is larger than minrelaytxfee')
-        assert_equal(node.getmempoolinfo()['minrelaytxfee'], Decimal('0.00001000'))
+        self.log.info("Check that mempoolminfee is larger than minrelaytxfee")
+        assert_equal(node.getmempoolinfo()["minrelaytxfee"], Decimal("0.00001000"))
         assert_greater_than(
-            node.getmempoolinfo()['mempoolminfee'], Decimal('0.00001000')
+            node.getmempoolinfo()["mempoolminfee"], Decimal("0.00001000")
         )
 
         # Deliberately tries to create a tx with a fee less than the minimum mempool fee to assert that it does not get added to the mempool
-        self.log.info('Create a mempool tx that will not pass mempoolminfee')
-        assert_raises_rpc_error(-26,"mempool min fee not met",miniwallet.send_self_transfer,from_node=node,fee_rate=relayfee,mempool_valid=False)
+        self.log.info("Create a mempool tx that will not pass mempoolminfee")
```

Comment:
since you introduced this code in the last commit, it would make more sense to just style it properly there & then not retouch in the subsequent commit